docs/frontend_utilisation_guide.md

```diff
@@ -93,35 +143,66 @@ document.getElementById("myElementId").appendChild(spsApplication.rootElement);
+```
+4) Create the `.env` file in the path you set in step 3 with the variable of your choice
+```bash
+WEBSOCKET_URL=ws://example.com/your/ws
+```
+5) Declare your environment variable where you instantiate your Scalable Pixel Streaming Frontend library 
 ```typescript
-import { Config, PixelStreaming, TextParameters } from '@epicgames-ps/lib-pixelstreamingfrontend-ue5.2';
+declare var WEBSOCKET_URL: string;
 ```
-2) Beneath the line where the `config` object was created call the `setTextSetting` method and pass in `TextParameters.SignallingServerUrl` and your websocket address as a string
+6) Make use of the `setTextSetting` method within the `config` instance to set the `TextParameters.SignallingServerUrl` to a variable that makes use of `WEBSOCKET_URL`
 ```typescript
-const config = new Config({ useUrlParams: true, initialSettings: { OfferToReceive: true, TimeoutIfIdle: true } });
-config.setTextSetting(TextParameters.SignallingServerUrl, "wss://your.websocket.url/ws")
+let webSocketAddress = WEBSOCKET_URL;
+if(webSocketAddress != ""){
+	config.setTextSetting(TextParameters.SignallingServerUrl, webSocketAddress)
+}
 ```
 
 ---
 ## Scalable Pixel Streaming Frontend customisation
-By default the Scalable Pixel Streaming Frontend contains all the requirements to connect to a Scalable Pixel Streaming signalling server making it an effective starting template for further customisation rather than starting from scratch. For further ways to utilise the the Pixel Streaming Frontend refer to the [Pixel Streaming Frontend documentation](https://github.com/EpicGames/PixelStreamingInfrastructure#readme).   
+By default the Scalable Pixel Streaming Frontend library contains all the requirements to connect to a Scalable Pixel Streaming signalling server making it an effective starting template for further customisation rather than starting from scratch. It is able to achieve this functionality through its consumption of the Epic Games Pixel Streaming Frontend. To learn more about further utilising the Epic Games Pixel Streaming Frontend documentation can be found [here](https://github.com/EpicGames/PixelStreamingInfrastructure#readme).   
 
 ### Scalable Pixel Streaming Frontend UI element customisation
-Further customisation of UI elements like Overlays or visual elements can be achieved by utilising the Pixel Streaming Frontend UI and extending its types. For further information on how to utilise the Epic Games Pixel Streaming Frontend UI refer to the [Pixel Streaming Frontend UI documentation](https://github.com/EpicGames/PixelStreamingInfrastructure#readme).
\ No newline at end of file
+Further customisation of UI elements like Overlays or visual elements can also be achieved by utilising the Pixel Streaming Frontend UI and extending its types. For further information on how to utilise the Epic Games Pixel Streaming Frontend UI refer to the [Pixel Streaming Frontend UI documentation](https://github.com/EpicGames/PixelStreamingInfrastructure#readme).
```

Comment:
does overlays need a capital here?

Reply:
Resolved in 4820831c542f48909f428f142606b7c814b6ede6